src/figure_factory.py

```diff
@@ -10,14 +10,14 @@ class FigureFactory:
     __kind_randomizer = FigureKindRandomizer()
 
     @staticmethod
-    def create(kind: FigureKind) -> Figure:
+    def create(kind: FigureKind, weight: int) -> Figure:
```

Comment:
Lets rename like "`board_weight`" in all place in `FigureFactory`

Reply:
in `create_random()` too

Reply:
done